src/main/java/uk/gov/hmcts/reform/et/syaapi/service/PdfMapperService.java

```diff
@@ -0,0 +1,325 @@
+package uk.gov.hmcts.reform.et.syaapi.service;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.et.common.model.ccd.Address;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.RespondentSumTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.ClaimantOtherType;
+import uk.gov.hmcts.et.common.model.ccd.types.NewEmploymentType;
+import uk.gov.hmcts.et.common.model.ccd.types.RepresentedTypeC;
+import uk.gov.hmcts.et.common.model.ccd.types.RespondentSumType;
+import uk.gov.hmcts.reform.et.syaapi.constants.PdfMapperConstants;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Maps Case Data attributes to fields within the PDF template.
+ * Inputs that are accepted from the ET1 form can then be mapped to
+ * the corresponding questions within the template PDF Provided
+ */
+@Service
+public class PdfMapperService {
+    private static final  Map<String, String> TITLES = Map.of(
+        "Mr", PdfMapperConstants.Q1_TITLE_MR,
+        "Mrs", PdfMapperConstants.Q1_TITLE_MRS,
+        "Miss", PdfMapperConstants.Q1_TITLE_MISS,
+        "Ms", PdfMapperConstants.Q1_TITLE_MS
+    );
+    private static final Map<String, String> TITLE_MAP = Map.of(
+        "Mr", "Mister",
+        "Mrs", "Mrs",
+        "Miss", "Mis",
+        "Ms", "Ms"
+    );
+    private static final String[] ADDRESS_PREFIX = {
+        "2.2",
+        "2.5 R2",
+        "2.7 R3",
+        "13 R4",
+        "13 R5"
+    };
+    private static final String REP_ADDRESS_PREFIX = "11.3 Representative's address:";
+    private static final String CLAIMANT_ADDRESS_PREFIX = "1.5";
+    private static final Integer MULTIPLE_RESPONDENTS = 2;
+    private static final Integer MAX_RESPONDENTS = 5;
```

Comment:
This should be an int.
Also if you consider your usage with greater than or equals
`           if (i >= MAX_RESPONDENTS) {`
aren't you in effect saying that the maximum number of respondents is 4 rather than 5?

Reply:
The array is 0-indexed so it's 5 respondents but the maximum index is 4. I felt like this makes it a bit easier to understand when reading.

Reply:
Ok got it my mistake. It should still be an int though - there's no need to use an object when a primitive will do